src/chocolatey.resources/helpers/functions/Install-ChocolateyPath.ps1

```diff
@@ -1,6 +1,6 @@
-# Copyright © 2017 Chocolatey Software, Inc.
-# Copyright © 2015 - 2017 RealDimensions Software, LLC
-# Copyright © 2011 - 2015 RealDimensions Software, LLC & original authors/contributors from https://github.com/chocolatey/chocolatey
+# Copyright � 2017 Chocolatey Software, Inc.
+# Copyright � 2015 - 2017 RealDimensions Software, LLC
+# Copyright � 2011 - 2015 RealDimensions Software, LLC & original authors/contributors from https://github.com/chocolatey/chocolatey

```

Comment:
It looks like you possibly changed line endings on this file. That will break it for authenticode signing.